packages/template-retail-react-app/app/pages/registration/index.jsx

```diff
@@ -38,11 +38,11 @@ const Registration = () => {
             password: data.password
         }
 
-        return register.mutateAsync(body, {
-            onError: () => {
-                form.setError('global', {type: 'manual', message: formatMessage(API_ERROR_MESSAGE)})
-            }
-        })
+        try {
+            await register.mutateAsync(body, {})
```

Comment:
```suggestion
            await register.mutateAsync(body)
```
Is the options object required if it's empty?